packages/table/src/table.tsx

```diff
@@ -440,11 +440,11 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
             const cType = child.type;
 
             if (typeof cType === "string") {
-                throw new Error(ERROR_MESSAGE);
+                console.warn(ERROR_MESSAGE);
             } else {
                 const isColumn = cType.prototype === Column.prototype || Column.prototype.isPrototypeOf(cType);
                 if (!isColumn) {
-                    throw new Error(ERROR_MESSAGE);
+                    console.warn(ERROR_MESSAGE);
```

Comment:
it's more of a warning; usually the code will still work. we might want to change the variable to `WARNING_MESSAGE`